### PR TITLE
I made a sprite

### PR DESCRIPTION
**Pull request by JhinHandaGay21**
## About The Pull Request
I spent the whole night creating a sprite jacket for all players and would like this jacket to be added to the game

## How This Contributes To The Skyrat Roleplay Experience
people will have more choice

## Changelog

:cl:
add: Added more things
imageadd: added some icons and images
code: changed some code
:cl:

### CODE REVIEW

File: code/modules/clothing/suits/jacket.dm [hunk 1]
```diff
@@ -9,6 +9,14 @@
...
 	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
 	species_exception = list(/datum/species/golem)
+	
+	/obj/item/clothing/suit/jacket
+	 name = "pink jacket"
+	 desc = "PINK PINK PINK"
+	 icon_state = "pink_jacket"
+     inhand_icon_state = "leatherjacket"
+	 body_parts_covered = CHEST|GROIN|ARMS
+	 armor = list(MELEE = 1, BULLET = 1, LASER = 1,ENERGY = 1, BOMB = 1, FIRE = 1, ACID = 1)
```
Comment (tf-4) [inline code/modules/clothing/suits/jacket.dm]:
```suggestion
/obj/item/clothing/suit/jacket
	name = "pink jacket"
	desc = "PINK PINK PINK"
	icon_state = "pink_jacket"
	inhand_icon_state = "leatherjacket"
	body_parts_covered = CHEST|GROIN|ARMS
	armor = list(MELEE = 1, BULLET = 1, LASER = 1,ENERGY = 1, BOMB = 1, FIRE = 1, ACID = 1)
```
Your indentation is off

File: code/modules/clothing/suits/jacket.dm [hunk 2]
```diff
@@ -9,6 +9,14 @@
 	cold_protection = CHEST|GROIN|ARMS
 	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
 	species_exception = list(/datum/species/golem)
+	
+	/obj/item/clothing/suit/jacket
```
Comment (tf-4) [inline code/modules/clothing/suits/jacket.dm]:
```suggestion
/obj/item/clothing/suit/jacket/pink
```
This needs to be a subtype

File: code/modules/clothing/suits/jacket.dm [hunk 3]
```diff
@@ -9,6 +9,14 @@
...
+	 name = "pink jacket"
+	 desc = "PINK PINK PINK"
+	 icon_state = "pink_jacket"
+     inhand_icon_state = "leatherjacket"
+	 body_parts_covered = CHEST|GROIN|ARMS
```
Comment (tf-4) [inline code/modules/clothing/suits/jacket.dm]:
```suggestion
```
This is the same as the parent

File: code/modules/clothing/suits/jacket.dm (hunk 1 above)
Comment (tf-4) [inline code/modules/clothing/suits/jacket.dm]:
```suggestion
```
Shouldn't be armoured.

File: code/modules/vending/clothesmate.dm [hunk 4]
```diff
@@ -77,6 +77,7 @@
 		/obj/item/clothing/suit/jacket = 2,
 		//obj/item/clothing/suit/hooded/wintercoat/custom = 2, SKYRAT EDIT - Moved to modular equivalent
 		/obj/item/clothing/suit/jacket/puffer/vest = 2,
+		/obj/item/clothing/suit/pink_jacket = 3,
```
Comment (tf-4) [inline code/modules/vending/clothesmate.dm]: move this to below this line here,
https://github.com/Skyrat-SS13/Skyrat-tg/blob/10a2c07de6e0b2d7fa20a40f70471d730f820563/modular_skyrat/modules/modular_vending/code/clothesmate.dm#L95

File: code/modules/clothing/suits/jacket.dm (hunk 2 above)
Comment (tf-4) [inline code/modules/clothing/suits/jacket.dm]: Move all this to `modular_skyrat/modules/customization/modules/clothing/suits/jacket.dm`. The folder is there but you just need to make a new file. Ensure the new file is ticked once you've made it.

**Review by tf-4 — CHANGES_REQUESTED** [review]

**Comment by tf-4** [comment]
You also need screenshots taken of the sprite in-game, posted in the body of the PR.

Poke me in discord if you need help with any part of this.

**Comment by Lukaster42** [comment]
death.

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
- Fix your changelog
- Modularize this as said
- Please confirm you actually tested this, seeing as you made this entirely out of webedits

File: код/модули/Одежда/Костюмы/jacket.dm [hunk 5]
```diff
@@ -0,0 +1,87 @@
+/obj/item/clothing/suit/jacket
```
Comment (Zonespace27) [inline код/модули/Одежда/Костюмы/jacket.dm]: не русский

File: код/модули/Одежда/Костюмы/jacket.dm [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+	species_exception = list(/datum/species/golem)
+	
+/obj/item/clothing/suit/jacket/pink_jacket
+	name = "pink jacket"
+	desc = "Pompadour not included."
+	icon_state = "pink_jacket"
+	inhand_icon_state = "pink_jacket"
+	
+
+/obj/item/clothing/suit/jacket/leather
```
Comment (Zonespace27) [inline код/модули/Одежда/Костюмы/jacket.dm]: ...none of these other jackets are used anywhere

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/modules/clothing/suits/jacket.dm [hunk 7]
```diff
@@ -10,6 +10,12 @@
 	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
 	species_exception = list(/datum/species/golem)
 
+/obj/item/clothing/suit/jacket/pink_jacket
+	name = "pink jacket"
+	desc = "PINK PINK PINK"
+	icon_state = "pink_jacket"
+	inhand_icon_state = "leatherjacket"
+
```
Comment (ghost) [inline code/modules/clothing/suits/jacket.dm:18]: >TG master file addition

**Comment by ghost** [comment]
Also, looking at the sprite via the icondiffbot, its just- eh?
There's minimal shading, and pink with gold accents is a combination that shouldn't see the light of day.
The detailing is confusing at best - why a helix on the back, if thats what thats meant to be? What's the dark colors in the front, an undershirt? The description isn't helpful either
Moreover, forced colors for one-off items is bad anyways. Greyscale jackets are or can be a thing, including in this jacket's shape

**Comment by ChungusGamer666** [comment]
chad pr

**Comment by Gandalf2k15** [comment]
Re-make a PR once you have completed this.

**Pull request closed without merging**